docs/administering/sniproxy.md

```diff
@@ -0,0 +1,146 @@
+# Sharing the HTTPS Port with sniproxy
+
+The purpose of this tutorial is to set up sniproxy so it’s possible to use regular https:// URLs for your main web server, like nginx or Apache, as well as Sandstorm, which has its own HTTPS handling. To make that work, they will need to share port 443.
+
+## Introduction
+
+The purpose of this tutorial is to set up sniproxy so it's possible to use Sandstorm-verified TLS encryption while coexisting with another web server that also uses TLS and share port 443 with it.
+
+The main reason is to allow users to connect with your Sandstorm instance in the standard HTTPS port (443) and keep using that port also for any other web apps.
+
+This action is going to change the BASE_URL of Sandstorm, presumably, so you'll end up erasing the OAuth configuration. To fix that, you will need to follow the "How do I log in, if there's a problem with logging in via the web?" section of [the FAQ](faq.md).
```

Comment:
I think this should be worded more conditionally; if a user is going from sandstorm facing the internet directly to moving it behind sniproxy, BASE_URL will not change. This seems to be making some unstated assumptions about the user's prior configuration.

We should probably have a section near the top that states explicitly what the starting point is.

Reply:
I think it might be fair to just pull this out entirely. You do make a good point, and the solution to that additional problem is elsewhere in the docs already.

Reply:
SGTM.